src/cuda-install.sh

```
@@ -0,0 +1,1002 @@
+    for code in "${codes[@]}"; do
+        printf "  %-4s - %s\n" "$code" "$(get_exit_code_description "$code")"
+    done
+
+    echo "═══════════════════════════════════════════════════════════════"
+    echo "Last exit code: $STATE_DIR/last_exit_code"
+}
+
+# 交互式确认
+confirm() {
+    local prompt="$1"
+    local default="${2:-N}"
```

Shellcheck (0.11.0)

[warning] default appears unused. Verify use (or export if used externally).

(SC2034)